internal/common/config/config.go

```diff
@@ -11,12 +10,8 @@ import (
 
 var rdsCaPath *string
 
-func clowderEnabled() bool {
-	return strings.ToLower(os.Getenv("CLOWDER_ENABLED")) != "false"
-}
-
 func init() {
-	if !clowderEnabled() || clowder.LoadedConfig.Database.RdsCa == nil {
+	if clowder.IsClowderEnabled() || clowder.LoadedConfig.Database.RdsCa == nil {
```

Comment:
should this be `!clowder.IsClowderEnabled()` ?

Reply:
Yep! Nice catch